Makefile

```diff
@@ -381,6 +381,7 @@ partialclean::
 	rm -f compilerlibs/ocamlopttoplevel.cmxa
 
 ocamlnat: compilerlibs/ocamlcommon.cmxa compilerlibs/ocamloptcomp.cmxa \
+    compilerlibs/ocamlbytecomp.cmxa \
```

Comment:
The rule for `ocamlnat` is very different in `Makefile.nt`, so different, actually, that I am not very confident it is still correct (I couldn't find any occurrence of the definition of the `NATTOPOBJS` variable used there)